roles/ocp_on_libvirt/README.md

```diff
@@ -20,4 +44,46 @@ Please add the following variables to the libvirt cluster configuration:
     bus: "0x86"
     slot: "0x00"
     function: "0x0"
+```
+
+## How to create a libvirt cluster based on Virtualmedia over Redfish
+
+### Example of inventory file
+
+```
+all:
+  children:
+    vm_host:
+      hosts:
+        libvirthost:
+          vbmc_user: MY_USER
+          vbmc_password: MY_PASSWORD
```

Comment:
is it `vbmc_password` or `vbmc_pass`?

Reply:
I still have this question in place

Reply:
You're right, Ramon. Thanks for the catch.